hw/ip/otbn/data/insns.yml

```diff
@@ -1777,8 +1777,8 @@ insns:
       - If `grs1_inc` is set, the value in `grs1` is incremented by the value WLEN/8 (one word).
       - If `grs2_inc` is set, the value in `grs2` is incremented by the value 1.
 
-      Only aligned (to WLEN sized regions) accesses are supported.
-      Any unaligned access will result in an error.
+      The memory address must be aligned to WLEN bytes.
+      Any address that is unaligned or is above the top of memory will result in an error (with error code `ErrCodeBadDataAddr`).
     decode: |
```

Comment:
Sorry should have made this clearer, we'll want a pseudo-code update here (or rather just below here github won't let me comment on lines that aren't part of the diff).

Reply:
Oh, right, now I understand. I've been avoiding touching that, since it was yanked wholesale from Stefan's simulator. I'll try to write something sensible.